uint32 timestamp_ms		# in milliseconds since system start
int8[10] key			# max. 10 characters as key / name
float32 value			# the value to send as debug output
float64 value1			# the value to send as debug output
float32 value2			# the value to send as debug output
float32 value3			# the value to send as debug output
float32 value4			# the value to send as debug output
float32 value5			# the value to send as debug output
float32 value6			# the value to send as debug output
float32 value7			# the value to send as debug output


